sdk/storage/Azure.Storage.Files.Shares/tests/BenchmarkTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Storage.Files.Shares.Models;
+using Azure.Storage.Files.Shares.Specialized;
+using Azure.Storage.Sas;
+using Azure.Storage.Test;
+using Azure.Storage.Test.Shared;
+using Moq;
+using NUnit.Framework;
+
+namespace Azure.Storage.Files.Shares.Tests
+{
+    public class BenchmarkTests : FileTestBase
```

Comment:
Why is this called benchmark test? This looks more like functional test (which we should have anyway).

For perf/benchmarking I suggest to add a scenario in https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/storage/Azure.Storage.Files.Shares/perf/Microsoft.Azure.Storage.File.Perf/Scenarios and use perf framework to run benchmarks.

Reply:
This might be helpful here https://github.com/Azure/azure-sdk-for-net/wiki/Writing-performance-tests-for-Client-libraries .

Reply:
Oh I was trying to just test a couple of scenarios for figuring out what the Share defaults for the transfer options should be, but forgot to delete this. Good to know on Perf though.

I couldn't really tell what the max was, though. Seemed like requests even up to 1 gigabyte succeeded and returned the entire contents inside the first request if set up to do so.